packages/create-mc-app/src/cli.ts

```diff
@@ -25,7 +25,12 @@ const run = () => {
   cli
     .command('[project-directory]')
     .usage(
-      '[project-directory]\n\n  Bootstraps a new Custom Application project using one of the predefined templates.'
+      '[project-directory]\n\n  Bootstraps a new Application project using one of the predefined templates.'
+    )
+    .option(
```

Comment:
We add a new parameter so developers can specify which type of application wants to create.